OsuPlayer.Wpf/Windows/MiniWindow.xaml.cs

```diff
@@ -93,8 +95,16 @@ private void Window_Loaded(object sender, RoutedEventArgs e)
                 Left = _screenWidth - ActualWidth - 20;
                 Top = _screenHeight - ActualHeight - 100;
             }
+            if (area != null && area.Length == 4)
+            {
+                _currentArea = new Rectangle(area[0], area[1], area[2], area[3]);
+            }
```

Comment:
Add an `else` here and initialize a default value: 
```cs
_currentArea = Screen.GetWorkingArea(new System.Drawing.Point((int)Left, (int)Top));
```

Reply:
added 5c1f431